tests/slow/src/test/scala/tests/feature/WorksheetCrossLspSuite.scala

```diff
@@ -2,12 +2,45 @@ package tests.feature
 
 import scala.meta.internal.metals.{BuildInfo => V}
 
-class Worksheet211LspSuite extends tests.BaseWorksheetLspSuite(V.scala211)
-class Worksheet3LspSuite extends tests.BaseWorksheetLspSuite(V.scala3) {
+abstract class Worksheet211LspSuite(
+    useVirtualDocuments: Boolean,
+    suiteNameSuffix: String
+) extends tests.BaseWorksheetLspSuite(
+      V.scala211,
+      useVirtualDocuments,
+      suiteNameSuffix
+    )
+class Worksheet211LspSaveToDiskSuite
```

Comment:
We can run `virtual-docs` as default and let's just add a single test suite (could be case really) that runs with `save-to-disk`. I don't think we need to replicate it for every Scala versions and does take a while to run it. We can even remove the `2.11` suite I think, since we did depracte it.